Pasfmt.dpk

```diff
@@ -3,6 +3,7 @@
 {$R *.res}
 {$R *.dres}
 {$IFDEF IMPLICITBUILDING This IFDEF should not be used by users}
+{pasfmt off}
```

Comment:
Also, the IDE removes this comment when regenerating the file. Maybe just put it at the top?

Reply:
We'll move it to the top and remove the `{pasfmt on}` line.